src/adcm_client/packer/naming_rules.py

```diff
@@ -123,7 +127,10 @@ def write_version(file, old_version, new_version):
     build_id = ''
     if git_data:
         check_version(version)
-        build_id = resolve_build_id(git_data, master_branches, timestamp)
+        build_id = resolve_build_id(git_data, master_branches, timestamp, timestamp_enabled)
         write_version(bundle.file, version, version + build_id)
+    else:
+        if timestamp_enabled:
+            build_id = build_id + '_' + timestamp
```

Comment:
by default build_id is empty string, a u sure that it is expected behavior for version to be like "_12344334534"

Reply:
i dont know if there are problem with rpm version specification (we use it for adcm)
https://docs.fedoraproject.org/en-US/packaging-guidelines/Versioning/